Collections.cs

```diff
@@ -245,6 +246,32 @@ public string GetPoliticianInfo(Politician politician)
             return $"{politician.Id}, {politician.FirstName}, {politician.LastName}, {politician.Age}, {politician.Position}";
         }
 
+        public void JsonForPolitician()
+        {
+            var PoliticianSalary = new List<int>() { 100, 500, 1000, 10000 };
```

Comment:
not quite what was expected here. 
please extend any of the existing classes that use JSON annotations with more complex properties e.g.:

public class Politician{
public List<string> Hobbies;
public List<Pet> Pets;
}

and then serialize/deserialize it.
the idea here is for you to work with more complex JSON objects